docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,109 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
 ## Getting started
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <Button>Hello World</Button>
+    </CssVarsProvider>
+  );
 }
+```
+
+### CssVarsProvider
+
+In the example above, you can see that the Button component is nested within `<CssVarsProvider />`.
+This is not required, but it unlocks a whole host of customization options powered by CSS variables.
+See [Using CSS variables](/joy-ui/customization/using-css-variables/) for more details.
+
+## Shared props
+
+Joy UI components are self-supporting and fully functional in isolation.
```

Comment:
I do not quite understand what this line means.

Reply:
If we keep this sentence on, I'm feeling like it would be best put it right below the first Getting started heading?! I get that at this current spot, it's a hook for the shared props stuff but it seems something we may want to share right away?

Reply:
Hmm, it's something that I original found in the Material UI Usage page, and copied it over to Base and then Joy. As I understand it, the idea is that every individual component can be used by itself in an app—it doesn't require the use of any other components to work correctly. In order words, you don't have to build a strictly Joy UI-only app in order to use a Joy component as a one-off in a project.

Reply:
this is accurate, it's also about the idea that you can add this inside an existing Bootstrap application, and you won't break everything nor will it break your component because they use box-sizing: box-content and not border-content.